### ISSUE TITLE
Supplementation feature is missing

### ISSUE THREAD

**Issue by LotekHeavy** [dossist/moinmoin-memodump#40]
It seems that the supplementation feature of the MoinMoin Wiki you're not supporting.

> supplementation_page  False   // if True, show a link to the supplementation page in the theme
> http://moinmo.in/HelpOnConfiguration/SupplementationPage

I know we could create an own menu (https://github.com/dossist/moinmoin-memodump/wiki/EditMenu) but IMHO it should be immediately visible on the front - with a icon like the edit button.

Not sure where excatly... but in this direction :)
![2014-09-30 11_33_10-protokolle_2014-09-23 - informatik](https://cloud.githubusercontent.com/assets/7978295/4455951/51c56c0a-4885-11e4-8d42-d02c0241a5ce.png)

**Comment by dossist** [comment]
You are right. I omitted a support for it when I first made the theme. Now it would be the time for it to be implemented.
